src/components/NavBar/NavigationBar.tsx

```diff
@@ -129,30 +135,36 @@ const ResponsiveAppBar: React.FC<Props> = ({ children = <></> }) => {
     };
 
     const pages = [
-        ["Clients", "/clients"],
```

Comment:
Can we not do this? I think screen-readers sometimes struggle to read all capitals, the text itself should be `Clients`. If you want it to be in all capitals (which I agree, I think it looks better!) then this should be a CSS-only change - using something like `text-transform: uppercase`

Reply:
Done